client/src/components/App/Content/UsersManagement/UsersManagement.tsx

```diff
@@ -51,6 +51,7 @@ const UsersManagement: React.FC = () => {
                 direction: cellNameSort.name !== cellName ? SortOrder.asc :
                            cellNameSort.direction === SortOrder.asc ? SortOrder.desc : SortOrder.asc
             });
+            setPage(1);
```

Comment:
Do we want to be back at page 1 when sorting as well? if so, this doesnt happen on user management so maybe they should act the same. Anyway, use default tab here

Reply:
Yes they both shoul do so checked with Laly, will delete this line

Reply:
and it is done in use management page, check useUsersManagement.ts